common/src/main/java/org/conscrypt/ConscryptEngineSocket.java

```diff
@@ -42,7 +42,7 @@
 /**
  * Implements crypto handling by delegating to {@link ConscryptEngine}.
  */
-final class ConscryptEngineSocket extends OpenSSLSocketImpl {
+class ConscryptEngineSocket extends OpenSSLSocketImpl {
```

Comment:
If you're going to remove `final`, you should add it to all the methods you can.

Reply:
Sure thing, done.